client/core/core_test.go

```diff
@@ -2638,9 +2658,12 @@ func TestTradeTracking(t *testing.T) {
 	if auth.AuditStamp != audit.Time {
 		t.Fatalf("audit time not set")
 	}
+	time.Sleep(250 * time.Millisecond) // let async tick do nothing
```

Comment:
Isn't this too short now? Can make that `2*time.Second` factor a package var for testing, or just drop this `Sleep`, since you're calling `tickAsset` right away anyways.

Reply:
There's the exception for `numMatches == 1`, and this sleep is my lazy way of letting the tick complete before setting the SwapConf a couple lines down.  Let me add some proper sync instead.